project/plugins.sbt

```diff
@@ -1,7 +1,7 @@
 logLevel := Level.Warn
 
-addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.19")
-addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.2")
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.8.2")
+libraryDependencies += "org.scala-js" %% "scalajs-env-selenium" % "0.2.0"
```

Comment:
Add comment about order linking to: https://github.com/scala-js/scala-js/issues/3164